zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala

```diff
@@ -69,6 +70,7 @@ trait CassandraSpanStoreFactory {self: App =>
       builder.withCredentials(cassandraUsername(), cassandraPassword())
     builder.withRetryPolicy(ZipkinRetryPolicy.INSTANCE)
     builder.withLoadBalancingPolicy(new TokenAwarePolicy(new LatencyAwarePolicy.Builder(new RoundRobinPolicy()).build()))
+    builder.withPoolingOptions(new PoolingOptions().setMaxConnectionsPerHost(HostDistance.REMOTE, cassandraMaxConnections()))
```

Comment:
the default load balancing is not ideal, we ran into issue with it and changed to 
```
    val lbPolicy = new TokenAwarePolicy(new LatencyAwarePolicy.Builder(new DCAwareRoundRobinPolicy(dc)).build())
```